packages/uploadthing/test/__test-helpers.ts

```diff
@@ -142,8 +118,35 @@ export const it = itBase.extend({
       insertFile: (file: any) => files.push(file),
       getFileByKey: (key: string) => files.find((f) => f.key === key),
     };
-    // prepend msw listeners to use db instance
     msw.use(
+      /**
+       * S3
+       */
+      http.post("https://bucket.s3.amazonaws.com", async ({ request }) => {
+        await callRequestSpy(request);
+        return new HttpResponse();
+      }),
+      http.put("https://bucket.s3.amazonaws.com/:key", async ({ request }) => {
+        await callRequestSpy(request);
+        return new HttpResponse(null, {
+          status: 204,
+          headers: { ETag: "abc123" },
+        });
+      }),
+      /**
+       * Static Assets
+       */
+      http.get("https://cdn.foo.com/:fileKey", async ({ request }) => {
+        await callRequestSpy(request);
+        return HttpResponse.text("Lorem ipsum doler sit amet");
+      }),
+      http.get("https://utfs.io/f/:key", async ({ request }) => {
+        await callRequestSpy(request);
+        return HttpResponse.text("Lorem ipsum doler sit amet");
+      }),
```

Comment:
moves here so that `import { it } from "vitest"` runs tests without __any__ mocks at all. `import { it } from "__test-helpers"` will run test with mocks